HISTORY.rst

```diff
@@ -2,11 +2,17 @@
 History
 =======
 
+0.5.0 (2023-??-??)
```

Comment:
We usually do this:

```suggestion
0.5.0 (UNRELEASED)
```

Reply:
OK.
I was planning to make a release right after merging this (I would update that before merging).